react/components/molecules/key-value/key-value.js

```diff
@@ -31,13 +33,19 @@ export class KeyValue extends PureComponent {
             iconHeight: undefined,
             iconWidth: undefined,
             iconStrokeWidth: undefined,
-            iconOnPress: undefined
+            onPress: undefined,
+            onButtonIconPress: undefined,
+            onLongPress: undefined
         };
     }
 
     render() {
         return (
-            <View style={styles.keyValue}>
+            <TouchableOpacity

```

Comment:
This being touchable or not should be controlled by a `enablePress` prop. We don't want to make all key-values touchable, just some of them.